CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## Release 1.6.7
+
+This is a minor patch release.
+
+New features and enhancements:
+- Bump ui-components to include decomposed Time Travel
+	[#2986](https://github.com/weaveworks/scope/pull/2986)
+- cheap probe connectedness api endpoint
+	[#2983](https://github.com/weaveworks/scope/pull/2983)
```

Comment:
both of these are really Weave Cloud features. Timetravel is not available in standalone scope, and the probe api endpoint is not invoked by the standalone UI.